sdk/cdn/arm-cdn/CHANGELOG.md

```diff
@@ -1,15 +1,136 @@
+  - Added Interface ProfilesMigrationCommitOptionalParams
+  - Added Interface ProfilesUpdateHeaders
+  - Added Interface ProfileUpgradeParameters
+  - Added Interface RoutesCreateHeaders
+  - Added Interface RoutesDeleteHeaders
+  - Added Interface RoutesUpdateHeaders
+  - Added Interface RulesCreateHeaders
+  - Added Interface RulesDeleteHeaders
+  - Added Interface RuleSetsDeleteHeaders
+  - Added Interface RulesUpdateHeaders
+  - Added Interface SecretsCreateHeaders
+  - Added Interface SecretsDeleteHeaders
+  - Added Interface SecurityPoliciesCreateHeaders
+  - Added Interface SecurityPoliciesDeleteHeaders
+  - Added Interface SecurityPoliciesPatchHeaders
+  - Added Interface UserAssignedIdentity
+  - Added Type Alias AfdProfilesCheckEndpointNameAvailabilityResponse
+  - Added Type Alias AfdProfilesUpgradeResponse
+  - Added Type Alias AfdProfilesValidateSecretResponse
+  - Added Type Alias CanMigrateDefaultSku
+  - Added Type Alias ManagedServiceIdentityType
+  - Added Type Alias MetricsGranularity
+  - Added Type Alias MetricsSeriesUnit
+  - Added Type Alias ProfilesCanMigrateResponse
+  - Added Type Alias ProfilesMigrateResponse
+  - Added Type Alias WafMetricsGranularity
+  - Added Type Alias WafMetricsSeriesUnit
+  - Interface AFDDomain has a new optional parameter extendedProperties
+  - Interface AFDDomainProperties has a new optional parameter extendedProperties
+  - Interface AzureFirstPartyManagedCertificateParameters has a new optional parameter certificateAuthority
+  - Interface AzureFirstPartyManagedCertificateParameters has a new optional parameter expirationDate
+  - Interface AzureFirstPartyManagedCertificateParameters has a new optional parameter secretSource
+  - Interface AzureFirstPartyManagedCertificateParameters has a new optional parameter subject
+  - Interface AzureFirstPartyManagedCertificateParameters has a new optional parameter subjectAlternativeNames
+  - Interface AzureFirstPartyManagedCertificateParameters has a new optional parameter thumbprint
+  - Interface CdnWebApplicationFirewallPolicy has a new optional parameter extendedProperties
+  - Interface Profile has a new optional parameter extendedProperties
+  - Interface Profile has a new optional parameter identity
+  - Interface ProfileUpdateParameters has a new optional parameter identity
+  - Added Enum KnownCanMigrateDefaultSku
+  - Added Enum KnownManagedServiceIdentityType
+  - Added Enum KnownMetricsGranularity
+  - Added Enum KnownMetricsSeriesUnit
+  - Added Enum KnownWafMetricsGranularity
+  - Added Enum KnownWafMetricsSeriesUnit
+  - Enum KnownProfileResourceState has a new value AbortingMigration
+  - Enum KnownProfileResourceState has a new value CommittingMigration
+  - Enum KnownProfileResourceState has a new value Migrated
+  - Enum KnownProfileResourceState has a new value Migrating
+  - Enum KnownProfileResourceState has a new value PendingMigrationCommit
 
-### Other Changes
+**Breaking Changes**
 
```

Comment:
swagger breaking pr?

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/23615

Reply:
accept this breaking as it is approved